src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -6,8 +6,9 @@
 using System.ServiceModel.Security;
 using System.Text;
 using Xunit;
+using Infrastructure.Common;
 
-public static class Https_ClientCredentialTypeTests
+public class Https_ClientCredentialTypeTests : ConditionalWcfTest
```

Comment:
I should have added the note that this behavior matches what the 'nameof' operator does and is probably the reason it was scoped to this behavior.  CoreFx has adopted nameof for its [ConditionalFact] usage.